config/db_tools.standalone.sample.yaml

```diff
@@ -1,130 +1,127 @@
+# unique connection.
+#connections:
+#    connection_one:
+#        # Connection URL for connecting.
+#        # Please refer to makinacorpus/db-query-builder or  documentation for more information.
+#        # Any URL built for doctrine/dbal usage should work.
+#        # URL is the sole mandatory parameter.
+#        # Complete list of accepted parameters follows.
+#        url: "pgsql://username:password@hostname:port?version=16.0&other_option=..."
+#        backup_binary: /usr/local/bin/vendor-one-dump
+#        backup_excluded_tables: ['table_one', 'table_two']
+#        backup_expiration_age: '1 month ago'
+#        backup_options: --no-table-lock
+#        backup_timeout: 2000
+#        restore_binary: /usr/local/bin/vendor-one-restore
+#        restore_options: --disable-triggers --other-option
+#        restore_timeout: 5000
+#        storage_directory: /path/to/storage
+#        storage_filename_strategy: datetime
+#    connection_two:
+#        url: "mysql://username:password@hostname:port?version=8.1&other_option=..."
+#        # ...
+
+# You can explicitely set which will be default connection in use when
+# none providen in the command line options. If you omit this configuration
+# value, then the first one in the list will be used.
+#default_connection: connection_one
 
 # Update this configuration if you want to look for anonymizers in a custom folder.
-# These are default paths that will always be registered even if you override
-# the setting and don't repeat them:
 #anonymizer_paths:
     #- ./src/Anonymization/Anonymizer'
 
+# Data anonymization configuration.
 anonymization:
     # From here you can proceed with manual file inclusion. Pathes can be
     # either relative or absolute. Relative paths are relative to the workdir
     # option if specified, or from this configuration file directory
     # otherwise.
+
+    # Each connection can have its configuration file. The configuration file
+    # structure is identical to the "table" section of the YAML, ommiting the
+    # "table" top-level key.
+
+    # See the "anonymizations.sample.yaml" in this folder for an example.
+
+    # If you want to load configuration from a yaml. If you have a single
+    # connection, you may omit the connection name and declare the file name
+    # this way:
+    #yaml: '%kernel.project_dir%/config/anonymizations.yaml'
+
+    # Alternatively if you have many connection, you can import a file for
+    # each connection using this syntax:
     yaml:
         connection_one: ./db_tools.anonymizer.connection_one.yaml
         # ... other connections ...
 
-    # Extra configuration options, if you don't want to split the anonymization
-    # configuration into multiple files, you can directly write it here.
+    # If you do not wish to split the configuration into tables, you can
+    # write all your table configuration directly here.
+    # Keys are connection names, values must follow the structure of the
```

Comment:
if there's only one connection, what's its name ?

does the 'connection' stage needed ? if not, may be you can omit this part and just redirect to the doc for a multiconnection setup